src/components/profile/ProfileModal.vue

```diff
@@ -0,0 +1,81 @@
+<script setup lang="ts">
```

Comment:
I would recommend to create `components/modals` folder and put it there. The same way how it is in extension.

Reply:
Okay, I'll do that now